polkadot/xcm/xcm-builder/xcm-payment-runtime-api/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "xcm-payment-runtime-api"
```

Comment:
What do you think about `xcm-fee-payment-runtime-api`?

Reply:
I like the suggested name because it is more explicit. On the other hand, what kind of payments (in the context of XCM) besides fees for delivering/executing messages exists?

Reply:
That's why I'm suggesting adding "fee" to the name, the only payments associated with XCMs are fees. Right now we have execution and delivery fees, but even if we wanted to add something else we would still call it a fee.

Reply:
When forming the name, we were guided by brevity, so we decided to omit the word `fee` since (at the moment) `fee payment` == `payment`.
As I mentioned above, I think a more complete (explicit) name will still look better.

Reply:
Crate renamed